src/bookshelf/rebar.config

```diff
@@ -10,26 +10,41 @@
     %% lager has to come first since we use its parse transform
     {lager, ".*",
         {git, "https://github.com/erlang-lager/lager",              {branch, "master"}}},
+    % this is here because of license scout issues
+    {base16, ".*",
+        {git, "https://github.com/goj/base16",                      {tag,    "1.0.0"}}},
     {cf, "",
         {git, "https://github.com/project-fifo/cf",                 {branch, "master"}}},
     {chef_secrets, ".*",
         {git, "https://github.com/chef/chef_secrets",               {branch, "main"}}},
+    % this is here because of license scout issues
+    {eini, ".*",
+        {git, "https://github.com/erlcloud/eini",                   {tag,    "1.2.6"}}},
```

Comment:
Could you please let us know where are we using this library in chef-server code.
Are we using this to resolve licence scout issues.

Reply:
Maybe transitive.  See comment above.

Reply:
license_scout changes have been removed from this PR, and put in the final PR for the Erlang upgrade.